### PR TITLE
Add repository info

### PR DESCRIPTION
**Pull request by leonardosnt**
Now it supports Github, Gitlab and Bitbucket

See https://github.com/yarnpkg/website/issues/754

### CODE REVIEW

File: src/formatPkg.js [hunk 1]
```diff
@@ -222,6 +239,65 @@ function getHomePage(homepage, repository) {
...
+  const result = repository.match(
+    /^https?:\/\/(?:www\.)?((?:github|gitlab|bitbucket)).((?:com|org))\/([^/]+)\/([^/]+)(\/.+)?$/
+  );
+
+  if (!result || result.length < 4) {
+    return null;
+  }
+
+  return {
+    host: `${result[1]}.${result[2]}`,
```
Comment (Haroenv) [inline src/formatPkg.js]: Here (and above as well) it would be prettier if we destructured `result`

Comment (leonardosnt) [reply src/formatPkg.js]: You mean like this:

```js
  const [, domain, domainTld, user, project, path = ''] = result;
  
  return {
    host: `${domain}.${domainTld}`,
    user,
    project,
    path,
  };
```

Comment (Haroenv) [reply src/formatPkg.js]: yep, that seems nicer to me

Comment (leonardosnt) [reply src/formatPkg.js]: > and above as well

Where? do you mean in another function? which one?

Comment (Haroenv) [reply src/formatPkg.js]: I meant in `getGithubInfo`, to which you referred in the comment

**Comment by Haroenv** [comment]
I'll give this a from 0 deployment, and it'll be completely reindexed in 8h or so

**Comment by leonardosnt** [comment]
Great! 🎉

**Pull request merged**